.phpunit.result.cache

```diff
@@ -0,0 +1 @@
reNonIdempotentMethods with data set #1":0,"Authorization\\Test\\TestCase\\Middleware\\UnauthorizedHandler\\RedirectHandlerTest::testHandleRedirectionIgnoreNonIdempotentMethods with data set #2":0,"Authorization\\Test\\TestCase\\Middleware\\UnauthorizedHandler\\RedirectHandlerTest::testHandleRedirectionIgnoreNonIdempotentMethods with data set #3":0,"Authorization\\Test\\TestCase\\Middleware\\UnauthorizedHandler\\RedirectHandlerTest::testHandleRedirectionIgnoreNonIdempotentMethods with data set #4":0,"Authorization\\Test\\TestCase\\Middleware\\UnauthorizedHandler\\RedirectHandlerTest::testHandleRedirectionIgnoreNonIdempotentMethods with data set #5":0,"Authorization\\Test\\TestCase\\Middleware\\UnauthorizedHandler\\RedirectHandlerTest::testHandleRedirectWithBasePath":0,"Authorization\\Test\\TestCase\\Middleware\\UnauthorizedHandler\\RedirectHandlerTest::testHandleException":0.001,"Authorization\\Test\\TestCase\\Policy\\Exception\\MissingPolicyExceptionTest::testConstructQueryInstance":0.001,"Authorization\\Test\\TestCase\\Policy\\Exception\\MissingPolicyExceptionTest::testConstructAnotherInstance":0,"Authorization\\Test\\TestCase\\Policy\\MapResolverTest::testGetPolicyClassName":0,"Authorization\\Test\\TestCase\\Policy\\MapResolverTest::testGetPolicyObject":0,"Authorization\\Test\\TestCase\\Policy\\MapResolverTest::testGetPolicyCallable":0,"Authorization\\Test\\TestCase\\Policy\\MapResolverTest::testMapMissingResource":0,"Authorization\\Test\\TestCase\\Policy\\MapResolverTest::testMapInvalidPolicy":0,"Authorization\\Test\\TestCase\\Policy\\MapResolverTest::testMapMissingPolicy":0,"Authorization\\Test\\TestCase\\Policy\\MapResolverTest::testGetPolicyPrimitive":0,"Authorization\\Test\\TestCase\\Policy\\MapResolverTest::testGetPolicyMissing":0,"Authorization\\Test\\TestCase\\Policy\\OrmResolverTest::testGetPolicyUnknownObject":0.004,"Authorization\\Test\\TestCase\\Policy\\OrmResolverTest::testGetPolicyUnknownEntity":0.001,"Authorization\\Test\\TestCase\\Policy\\OrmResolverTest::testGetPolicyDefinedEntity":0,"Authorization\\Test\\TestCase\\Policy\\OrmResolverTest::testGetPolicyDefinedPluginEntityAppOveride":0,"Authorization\\Test\\TestCase\\Policy\\OrmResolverTest::testGetPolicyDefinedPluginEntityPluginOveride":0.001,"Authorization\\Test\\TestCase\\Policy\\OrmResolverTest::testGetPolicyDefinedPluginEntity":0.001,"Authorization\\Test\\TestCase\\Policy\\OrmResolverTest::testGetPolicyDefinedTable":0.001,"Authorization\\Test\\TestCase\\Policy\\OrmResolverTest::testGetPolicyQueryForDefinedTable":0.001,"Authorization\\Test\\TestCase\\Policy\\OrmResolverTest::testGetPolicyUnknownTable":0.002,"Authorization\\Test\\TestCase\\Policy\\ResolverCollectionTest::testEmptyCollection":0,"Authorization\\Test\\TestCase\\Policy\\ResolverCollectionTest::testMissingPolicy":0.001,"Authorization\\Test\\TestCase\\Policy\\ResolverCollectionTest::testGetPolicy":0,"Authorization\\Test\\TestCase\\Middleware\\RequestAuthorizationMiddlewareTest::testUnauthorizedHandlerSuppress":0}}
```

Comment:
Was this file pushed on purpose?

Reply:
Doh.. no, no idea how that got it, will clean it up.

Reply:
.gitattributes doesnt contain the `.phpunit.result.cache` file yet.

Reply:
True, but I almost always use `git commit -a` to only commit modifications to existing files only.